<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #B7B89F;
}

QGroupBox {
    background-color: white;
    border: 1px solid #cccccc; /* Garis pinggir abu halus */
    border-radius: 10px;       /* Sudut membulat */
    margin-top: 20px;          /* Jarak buat judul */
}

/* Judul GroupBox */
QGroupBox::title {
    subcontrol-origin: margin;
    left: 15px;
    padding: 0 5px;
    color: #777C6D;
    font-weight: bold;
}

/* 3. Inputan */
QLineEdit, QComboBox, QSpinBox {
    background-color: #ffffff;
    border: 1px solid #aaaaaa;
    border-radius: 5px;
    padding: 5px;
    selection-background-color: #2980b9;
}

/* 4. Tabel */
QTableWidget {
    background-color: white;
    gridline-color: #dddddd;
    border: 1px solid #cccccc;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupSearching">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>CariPenerbangan</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sistem Booking R4KIT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputCari">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ketik kota tujuan...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCari">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #777C6D; 
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #7B542F;
}</string>
       </property>
       <property name="text">
        <string>Cari Tiket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboWaktu">
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Semua Waktu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pagi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Siang</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Malam</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tabelPenerbangan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow {
    background-color: #E0F7FA; 
}</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Kode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tujuan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Maskapai</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jam</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kursi</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBooking">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Formulir Pemesanan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblInfoPesawat">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pilih Penerbangan dulu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputNama">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Nama Penumpang (Gunakan _ spasi)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGacha">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #777C6D; 
    color: white;
    font-size: 14pt;
	 border-radius: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #7B542F;
}</string>
       </property>
       <property name="text">
        <string>Gacha Kursi!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblHasilKursi">
       <property name="font">
        <font>
         <family>Ravie</family>
         <pointsize>19</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
    background-color: #999999;
}</string>
       </property>
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
